cards.js

```diff
@@ -16,8 +16,45 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
-
+function buildDeck(values, suits){
 // Put some code in here!
+deck =[]
+for(var v = 0; v < values.length; v++){
+  for(var s = 0; s < suits.length; s++){
+    val = values[v];
+    suit = suits[s];
+      var card = {
+        cardface: val + " of " + suit,
+        value: val,
+        suit: suit,
+        strength: v
```

Comment:
Does the logic on line 52 work for all cases without translating strength to an integer when you store it on this object? ("J" -> 11, for example)